src/test/operators/table_scan_between_test.cpp

```diff
@@ -57,50 +57,91 @@ class TableScanBetweenTest : public TypedOperatorBaseTest {
     _data_table_wrapper = std::make_shared<TableWrapper>(data_table);
     _data_table_wrapper->execute();
   }
+
+  void _test_between_boundaries(std::vector<std::tuple<AllTypeVariant, AllTypeVariant, std::vector<int>>>& tests,
+                                bool left_inclusive = true, bool right_inclusive = true) {
+    const auto& [data_type, encoding, nullable] = GetParam();
+    std::ignore = encoding;
+    resolve_data_type(data_type, [&, nullable = nullable](const auto type) {
```

Comment:
We are not sure what the ``nullable = nullable`` does. We think you could use this to rename the variable, but here everything stays the same?

Reply:
have a look at line 38: `nullable=nullable` is a dirty hack to work around C++ defect 2313.
I have just copied this code and have not spent any further time questioning it.

Reply:
It seems like there is still no proper solution: http://www.open-std.org/jtc1/sc22/wg21/docs/cwg_defects.html